model_params:
  name: 'ReconVAE'
  latent_dim: 128
  obs_encoding_size: 1024
  goal_encoding_size: 1024
  context_size: 0
  waypoint_spacing: 5
  len_traj_pred: 5


data_params:
  train_batch_size: 64
  val_batch_size:  64
  patch_size: 64
  num_workers: 0
  context_size: 0
  waypoint_spacing: 5
  len_traj_pred: 5


exp_params:
  LR: 0.0005
  weight_decay: 0.0
  scheduler_gamma: 0.95
  kld_weight: 0.00025
  manual_seed: 1265
  distance_loss_weight: 0.5

trainer_params:
  accelerator: 'gpu'
  devices: 1
  max_epochs: 10

logging_params:
  save_dir: "logs/"
  name: "ReconVAE"
  
